webpack.config.development.js

```diff
@@ -8,7 +8,13 @@ config.devServer = {
     ]
   },
   hot: true,
-  inline: true
+  inline: true,
+  proxy: {
+    '/api': {
+      changeOrigin: true,
+      target: 'http://localhost:3000'
```

Comment:
It think it's used pretty much in all of our webpack configs. But I'll move it into its own PR.